ingest/workflow/snakemake_rules/split_serotypes.smk

```diff
@@ -0,0 +1,43 @@
+"""
+This part of the workflow handles splitting the data by serotype either based on the 
+NCBI metadata or Nextclade dataset. Could use both if necessary to cross-validate.
+
+    metadata = "data/metadata.tsv"
+    sequences = "data/sequences.fasta"
+
+This will produce output files as
+
+    metadata_{serotype} = "results/metadata_{serotype}.tsv"
+    sequences_{serotype} = "results/sequences_{serotype}.fasta"
+
+Parameters are expected to be defined in `config.transform`.
+"""
+
+rule split_by_ncbi_serotype:
+  """
+  Split the data by serotype based on the NCBI metadata.
+  """
```

Comment:
Minor request: Can you use the 4-space indentation throughout this new rules file to match the indentation used elsewhere in our workflows?

Reply:
Gah, sorry. I need to switch the default spacing in my code editor 😆 Fixed in 678fa5a7329fe989cbdb36217dd02b47cbe4448f